scripts/ingests/ingest_utils.py

```diff
@@ -0,0 +1,782 @@
+"""
+Utils functions for use in ingests
+"""
+from scripts.ingests.utils import *
+import logging
+import numpy as np
+import numpy.ma as ma
+from astropy.table import Table, unique
+import re
+import dateutil
+import pandas as pd
+from sqlalchemy import func
+
+logger = logging.getLogger('SIMPLE')
+
+
+# SOURCES
+def ingest_sources(db, sources, ras, decs, references, comments=None, epochs=None,
+                   equinoxes=None, raise_error=True):
+    """
+    Script to ingest sources
+
+    Parameters
+    ----------
+    db: astrodbkit2.astrodb.Database
+        Database object created by astrodbkit2
+    sources: list of strings
```

Comment:
Love the type hinting! However, think the syntax is generally: 
```suggestion
    sources: list[str]
```
for example